en/installation.rst

```diff
@@ -82,7 +82,7 @@ command:
 
 .. code-block:: bash
 
-    composer create-project --prefer-dist cakephp/app:4.* my_app_name
+    composer create-project --prefer-dist 'cakephp/app:^4' my_app_name
```

Comment:
The wildcard indeed creates quite a few issues on CLI systems. At least without the quotes.

Note: With `^` the quoting is not necessary anymore.
And if one wanted to specifically  pin it down to a minor, `cakephp/app:~4.0.0` should be used.

    composer create-project --prefer-dist cakephp/app:~4.0.0 my_app_name

Reply:
Its updated 😉